source/_integrations/garadget.markdown

```diff
@@ -44,14 +43,6 @@ covers:
           description: This is the device id from your Garadget portal.
           required: true
           type: string
-        username:
-          description: Your Garadget account username.
-          required: true
-          type: string
-        password:
-          description: Your Garadget account password.
-          required: true
-          type: string
```

Comment:
Our documentation reflects the codebase.

Did you open a PR to adjust the integration?

Reply:
To be honest, not sure. (I'm a little bit of a noob)
I found a solution for the cover not working, thought to update the docs so it doesn't get lost in the forums and it's ready available from the front of the page.
Please if I did something wrong accept my apologies, I'll try fix it for any other I update in the future

Reply:
I think we can leave the section on how to obtain the access token, that is really helpful!

But for the removal of configuration parameters, the Home Assistant integration code needs to be adjusted. (Suggestion: Raise an issue for that matter on the core repository?)